newrelic/data_source_newrelic_entity.go

```diff
@@ -28,21 +27,19 @@ func dataSourceNewRelicEntity() *schema.Resource {
 				Description: "Ignore case when searching the entity name.",
 			},
 			"type": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				Computed:     true,
-				Description:  "The entity's type. Valid values are APPLICATION, DASHBOARD, HOST, MONITOR, and WORKLOAD.",
-				ValidateFunc: validation.StringInSlice([]string{"APPLICATION", "DASHBOARD", "HOST", "MONITOR", "WORKLOAD"}, true),
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The entity's type. Valid values are APPLICATION, DASHBOARD, HOST, MONITOR,SERVICE and WORKLOAD.",
```

Comment:
Do we not have more valid types than this now? Also missing a space after the comma for `MONITOR,`.